includes/autoload_namespaces.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Default autoloader namespace configuration
+ *
+ * @package admin
+ * @copyright Copyright 2003-2014 Zen Cart Development Team
+ * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version $Id:  $
+ */
+
+// ensure library directory constants are set to rational defaults
+if (!defined('DIR_ADMIN_LIBRARY')) {
+  define('DIR_ADMIN_LIBRARY', DIR_FS_ADMIN . DIR_WS_INCLUDES . 'library/');
+}
+if (!defined('DIR_CATALOG_LIBRARY')) {
+  define('DIR_CATALOG_LIBRARY', DIR_FS_CATALOG . DIR_WS_INCLUDES . 'library/');
+}
+
+/**
+ * An array of namespace => basedir configurations
+ */
+return array(
+    '\Aura\Web' => DIR_CATALOG_LIBRARY . 'aura/web/src',
+    '\Aura\Di' => DIR_CATALOG_LIBRARY . 'aura/di/src',
+    'ZenCart\Platform' => DIR_CATALOG_LIBRARY. 'zencart/platform/src/',
+    'ZenCart\Paginator' => DIR_CATALOG_LIBRARY. 'zencart/paginator/src/',
+    'ZenCart\ListingBox' => DIR_CATALOG_LIBRARY. 'zencart/listingBox/src/',
```

Comment:
I guess this speaks to a bigger picture. 
We currently have two library directories, one in admin and one in catalog. I don't think we need both. One library under the main directory should be enough.

Given that,  the intention was that `platform' stuff should be code used by both admin and catalog,  as such paginator classes should be under the platform namespace. listingBox stuff is only used by catalog, so should not be under platform.